### PR TITLE
Add steward home page

### PR DESCRIPTION
**Pull request by forevermatt**
### Added
- Add home page for stewards
- Redirect stewards to their own home page

### Fixed
- Fix type error in money helper's `convertToCents()`

### CODE REVIEW

File: src/pages/customer/claims.svelte [hunk 1]
```diff
@@ -29,6 +30,8 @@ $: accountablePersons = [...policyMemberOptions, ...dependentOptions] as Account
 onMount(() => {
   loadClaims()
 })
+
+const onGotoClaim = (event) => $goto(`/customer/claims/${event.detail}`)
```
Comment (hobbitronics) [inline src/pages/customer/claims.svelte:34]:
```suggestion
const onGotoClaim = (event) => $goto(customerClaim(event.detail))
```

File: src/pages/steward/home.svelte [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import { itemsByPolicyId, loadItems } from 'data/items'
+import { loadMembersOfPolicy, membersByPolicyId } from 'data/policy-members'
+import { goto } from '@roxi/routify'
+import { Page } from '@silintl/ui-components'
+
+let claimsAwaitingSteward: Claim[]
+
+$: $claimsInitialized || loadClaims()
+$: claimsAwaitingSteward = $claims.filter(isAwaitingSteward)
+$: claimsAwaitingSteward.map((claim) => claim.policy_id).forEach(loadDataOnce)
```
Comment (hobbitronics) [inline src/pages/steward/home.svelte:15]: No need to change, but would this do the same thing? I am just curious if there is a difference or benefit.
```
claimsAwaitingSteward.forEach((claim) => loadDataOnce(claim.policy_id))
```

File: src/pages/steward/home.svelte [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  }
+}
+const onGotoClaim = (event) => $goto(`/steward/claims/${event.detail}`)
+</script>
+
+<style>
+</style>
+
+<Page layout="grid">
+  <Row cols={'12'}>
```
Comment (hobbitronics) [inline src/pages/steward/home.svelte:46]: Totally optional but I thought you made a similar suggestion to me.
```suggestion
  <Row cols="12">
```

File: src/pages/steward/home.svelte (hunk 2 above)
Comment (forevermatt) [reply src/pages/steward/home.svelte:15]: I think that would do the same thing, yeah

File: src/pages/steward/home.svelte [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+import { ClaimCards, Row } from 'components/'
```
Comment (hobbitronics) [inline src/pages/steward/home.svelte:2]: I think here lies the issue '/'

**Pull request merged**